Dockerfile.release

```diff
@@ -1,8 +1,12 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
jfyi - this was bumped to `22.04`

Reply:
cheers - just testing but no luck. Still throwing things at GHA to see what sticks in my fork. 
I'll close this PR until I'm done to avoid spamming the actions tab here too.